src/documentation/search/extracts.test.ts

```diff
@@ -62,3 +62,19 @@ describe("contextExtracts", () => {
     ]);
   });
 });
+
+describe("sortByStart", () => {
+  it("works", () => {
```

Comment:
Should we provide a better name for this test?

Reply:
9d67c7f